pvlib/singlediode.py

```diff
@@ -192,6 +192,14 @@ def bishop88_i_from_v(voltage, photocurrent, saturation_current,
     nNsVth : numeric
         product of diode ideality factor (n), number of series cells (Ns), and
         thermal voltage (Vth = k_b * T / q_e) in volts [V]
+    d2mutau : numeric
```

Comment:
"numeric, default 0" is the preferred style I think, but I'm certain that pvlib isn't consistent in this usage.  @wholmgren ?

Reply:
Yes, though I see it's not documented in the contributing guide.